src/cli/task.rs

```diff
@@ -9,6 +10,9 @@ use itertools::Itertools;
 use rattler_conda_types::Platform;
 use std::collections::HashSet;
 use std::error::Error;
+use std::fmt::Write as _;
```

Comment:
```suggestion
```

Reply:
deleting this gives out error though, i had to add this since the line below has `use std::io::{stdout, Write};`  is there an alternative?

```
08 | ... _ = write!(output, "{}, ", name.fancy_display());
    |         -------^^^^^^------------------------------- method not found in `String`
   --> /rustc/25ef9e3d85d934b27d9dada2f9dd52b1dc63bb04\library\core\src\fmt\mod.rs:194:8
    |
    = note: the method is available for `std::string::String` here    
    |
note: must implement `io::Write`, `fmt::Write`, or have a `write_fmt` method

```

Reply:
Check comment below.